app/controllers/application_controller.rb

```diff
@@ -1,13 +1,21 @@
 class ApplicationController < ActionController::Base
   include Pundit
   protect_from_forgery with: :exception, prepend: true
-  #before_action :authenticate_user!
+  before_action :authenticate_user!, if: :devise_controller?
   before_action :configure_permitted_parameters, if: :devise_controller?
 
   protected
 
   def configure_permitted_parameters
-    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :email, :password, :image, :type) }
+    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :email, :password, :image, :type ) }
   # devise_parameter_sanitizer.permit(:account_update) { |u| u.permit(:name, :email, :password, :current_password) }
   end
+
+  def after_sign_in_path_for(resource)
+    if current_user.type == "Admin"
```

Comment:
can be a single line `current_user.path == 'admin' ? plans_path : subscriptions_path`

Reply:
Done.